index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="tech-title">
+    <img class="title-decoration" src="assets/triangle.svg" alt="triangle" >
+    <p>Front-End Tools</p>
+  </div>
```

Comment:
There is a simpler way to do this, we try to use html only for really important structural elements, decoration elements like this image can be added throught CSS, use a ::before css pseudoclass to add this. With that change you could delete the div and the p tag and emphasize your title with html also adding it a header title that goes better with the title, like and h1, h2, h3 ......